daml/Marketplace/Distribution/Bidding/Model.daml

```diff
@@ -34,55 +29,19 @@ template Bid
   with
     operator : Party
     provider : Party
-    issuer : Party
     customer : Party
+    issuer : Party
     auctionId : Text
     assetId : Id
     details : Details
+    publish : Bool
```

Comment:
Remove this from here - this should create the `PublishedBid` contract when passed into a choice with `True`. Need to think about how to make the `PublishedBid` contracts visible to all `bidders / investors`